gcexport.py

```diff
@@ -1257,6 +1264,14 @@ def main(argv):
             print('[', actvty['activityId'], '] ', sep='')
             continue
 
+        # Action: Filtered out by typeID
+        if action == 'f':
+            # Display which entry we're skipping.
+            print('Filtering out due to typeID, {} not in {}   : Garmin Connect activity '.format(actvty['activityType']['typeId'], type_filter), end='')
+            print('(', current_index, '/', len(action_list), ') ', sep='', end='')
+            print('[', actvty['activityId'], ']', sep='')
```

Comment:
I would make it more compact by using f-strings.

```suggestion
            print(f'Filtering out due to typeID, {actvty['activityType']['typeId']} not in {type_filter}   : Garmin Connect activity ', end='')
            print(f'({current_index}/{len(action_list)}) [{actvty['activityId']}]')
```

Note: f-strings were introduced in Python 3.6, however, Python 3.5 is [no longer supported](https://endoflife.date/python).

Reply:
Agreed that this is nicer, but I just used the same formatting we're using for the existing skip/excluding outputs -- maybe we want to clean them all up together (in a separate PR?)

e.g.
https://github.com/pe-st/garmin-connect-export/blob/master/gcexport.py#L1245

Reply:
Fair point. I'm not sure if we have an official LTS policy, but I'd say the official PSF timeline is a benchmark.
@pe-st what are your thoughts on this?

Reply:
The lowest supported Python version was never defined explicitly. IMHO the script should be easy to be used, so that for common systems that come with python installed the installed python should be enough

- macOS Monterey has 3.8 (needs the Commandline Developer Tools though); macOS Catalina had 3.7
- Debian/Raspbian Bullseye has 3.9, Buster had 3.7
- Ubuntu 20 LTS has 3.8

So I'd say if we limit the features used to the [oldest active](https://devguide.python.org/#status-of-python-branches) version (currently 3.7), all users with a moderately current system should have no problems running the script.

Would a definition like that suit you?

Reply:
Absolutely!

I will start to convert some legacy code then. Starting with the f-strings to make it more compact.
Thanks for the comment!

Reply:
@joetimmerman I've just opened #78, I think it's okay to go with the suggestion above ☝️